android/cli/commands/_build.js

```diff
@@ -35,8 +35,8 @@ var ADB = require('node-titanium-sdk/lib/adb'),
 	temp = require('temp'),
 	SymbolLoader = require('appc-aar-tools').SymbolLoader,
 	SymbolWriter = require('appc-aar-tools').SymbolWriter,
-	ti = require('titanium-sdk'),
-	tiappxml = require('titanium-sdk/lib/tiappxml'),
+	ti = require('node-titanium-sdk'),
+	tiappxml = require('node-titanium-sdk/lib/tiappxml'),
```

Comment:
Can we please scan **all** usages for this require-issue? This is the 3rd commit where we need to manually adjust the path after initially changing it. @cb1kenobi ?